bench/Gimlet.adoc

```diff
@@ -0,0 +1,87 @@
+= Running OPTE Benchmarks on a Gimlet (pair)
+
+The kernel module benchmarks can be run on a pair of lab gimlets connected over their `cxgbe` interfaces.
+Today, SN9/SN14 are set up in this configuration.
+
+== Image construction
+When https://github.com/oxidecomputer/helios[building a helios image], we need to include some extra packages and pull in the latest (host-installed) version of OPTE.
+The below patch adds these packages behind a conditional flag:
```

Comment:
Is there a PR to just add this patch directly to helios? Alternatively seems like you could use the extra package functionality added in https://github.com/oxidecomputer/helios/pull/156.

Reply:
Sure, I'll open a PR for it in the morning -- I didn't want to interfere with the canonical build config, but someone else will likely find it useful.

Reply:
@luqmana I've just reworked the instructions in [1271e2e](https://github.com/oxidecomputer/opte/pull/486/commits/1271e2e1444948520210c995b5e13758db742a3a) to make use of `-F extra_packages+=x` instead, since it's only ~3 extra packages. I'll close oxidecomputer/helios#163.